src/daemon/config.rs

```diff
@@ -249,7 +249,9 @@ pub struct Config {
     #[serde(default = "ConfigDefaults::ca_refresh_seconds", alias = "ca_refresh")]
     pub ca_refresh_seconds: u32,
 
-    pub suspend_child_after_inactive_hours: Option<i64>,
+    #[serde(skip)]
+    suspend_child_after_inactive_seconds: Option<i64>,
```

Comment:
Why both hours and seconds? Is it because hours has already been serialized by some deployments of Krill and thus must continue to be supported?

Reply:
no, it's because I need to use seconds internally but wanted let users configure hours. We are not very consistent in time units in the configuration file. I think this is okay, because it depends on the context. In this case you really should use multiple hours and then the math using seconds gets annoying.. (unless you dream DNS I guess)

Reply:
Right, but you can express hours internally as seconds and then not store both? (and use `Duration` to handle the "math"?)

Reply:
Then I need custom deserialization when parsing the config file, I just figured this was less complicated.

Reply:
I would be nice to let users use notations like: 1s, 5m, 2h etc but it requires more work and we should then also review the other parameters. And probably remember for each attribute what the default unit should be for existing configurations where it is not specified.

Reply:
I wasn't referring to end user configuration, just internal representation (but yes it would be nice - add it to the todo list :-))

Reply:
Note that the 'seconds' field is not (de)serialized - so users can't set it

Reply:
Yes I realize that. I am only referring to having a single representation of this value internally in the Krill config struct.